prime-router/build.gradle.kts

```diff
@@ -328,6 +333,7 @@ dependencies {
     implementation("com.github.kayr:fuzzy-csv:1.6.48")
     implementation("org.commonmark:commonmark:0.17.2")
     implementation("com.google.guava:guava:30.1.1-jre")
+    implementation("com.helger.as2:as2-lib:4.7.1")
 
```

Comment:
The new AS2 library. Still be developed, but appears to only person who maintains it.